== SUBMITION (2019-05-30 00:29:10) 
-- CODE:
from numpy import * 
v = array(eval(input("Vetores: ")))

i = 0
j = 0

while(i < size(v)):
	if(v[i] >= 0):
		j = j + 1
	i = i + 1
	
k = 0 
l = 0

v2 = zeros(j, dtype = int)
while(k < size(v)):
	if(v[k] >= 0):
		v2[1] = v[k]
		l = l + 1
	k = k + 1
print(v2)
-- EXECUTION TIME:
0.593579
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1758/1183/main.py", line 18, in <module>
    v2[1] = v[k]
IndexError: index 1 is out of bounds for axis 0 with size 1

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-05-30 00:29:29) 
-- CODE:
from numpy import * 
v = array(eval(input("Vetores: ")))

i = 0
j = 0

while(i < size(v)):
	if(v[i] >= 0):
		j = j + 1
	i = i + 1
	
k = 0 
l = 0

v2 = zeros(j, dtype = int)
while(k < size(v)):
	if(v[k] >= 0):
		v2[l] = v[k]
		l = l + 1
	k = k + 1
print(v2)
-- EXECUTION TIME:
0.552616
-- TEST CASE 1:
---- input:
[-1,-2,-3,-4,-5,-6,-7,-8,-9,-10,1]
---- correct output:
[1]
---- user output:
[1]
-- TEST CASE 2:
---- input:
[2,-1,3,-4,5]
---- correct output:
[2 3 5]
---- user output:
[2 3 5]
-- TEST CASE 3:
---- input:
[-1,18,32,14,25,-6]
---- correct output:
[18 32 14 25]
---- user output:
[18 32 14 25]
-- GRADE:
100%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
